Makefile

```diff
@@ -20,3 +23,14 @@ docker-build-eno-reconciler:
 		--file docker/$(ENO_RECONCILER_IMAGE_NAME)/Dockerfile \
 		--tag $(REGISTRY)/$(ENO_RECONCILER_IMAGE_NAME):$(ENO_RECONCILER_IMAGE_VERSION) .
 	docker push $(REGISTRY)/$(ENO_RECONCILER_IMAGE_NAME):$(ENO_RECONCILER_IMAGE_VERSION)
+
+# Run tests with increased timeout for the reconciliation controller
+.PHONY: test-reconciliation
+test-reconciliation:
```

Comment:
I think this target should be `test` and should run all of the tests using the higher timeout of 5min